src/BulkAppointments/AppointmentForm.js

```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
   }
 
   if (!values.appointmentType) {
-    formErrors.push({ targetName: 'appointmentType', text: 'Appointment type is required' })
+    formErrors.push({ targetName: 'appointmentType', text: 'Select appointment type' })
   }
 
   if (!values.location) {
-    formErrors.push({ targetName: 'location', text: 'Location is required' })
+    formErrors.push({ targetName: 'location', text: 'Select location' })
   }
 
   if (!values.date) {
     formErrors.push({ targetName: 'date', text: 'Date is required' })
   }
 
   if (isToday && moment(values.startTime).isBefore(now)) {
-    formErrors.push({ targetName: 'startTime', text: "Start time shouldn't be in the past" })
+    formErrors.push({ targetName: 'startTime', text: 'Start time must be in the future' })
   }
 
   if (isToday && moment(values.endTime).isBefore(now)) {
-    formErrors.push({ targetName: 'endTime', text: "End time shouldn't be in the past" })
+    formErrors.push({ targetName: 'endTime', text: 'End time must be in the future' })
   }
 
   const DATE_TIME_FORMAT_SPEC = 'YYYY-MM-DDTHH:mm:ss'
 
   if (
     moment(values.endTime, DATE_TIME_FORMAT_SPEC).isBefore(moment(values.startTime, DATE_TIME_FORMAT_SPEC), 'minute')
   ) {
-    formErrors.push({ targetName: 'startTime', text: "Start time should't be after End time" })
-    formErrors.push({ targetName: 'endTime', text: "End time shouldn't be before Start time" })
+    formErrors.push({ targetName: 'startTime', text: 'The start time must be before the end time' })
```

Comment:
you haven't got `The` at the start of any of the other messages, so not needed here.
It is also odd that you are indicating that both fields are in error, whereas it is likely that just the end time is at fault.  I would have just highlighted the end time.

Reply:
The following text has been supplied by Clare.
'The start time must be before the end time’
’The end time must be after the start time

Reply:
You don't need both conditions if you reverse the logic so:
```
const isNotAfter = ! moment(values.endTime, DATE_TIME_FORMAT_SPEC).isAfter(
    moment(values.startTime, DATE_TIME_FORMAT_SPEC),
    'minute'
  )
```
although then it becomes clear that the variable should be called `endNotLaterThanStart` or similar